op-challenger/scripts/alphabet/init_game.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SOURCE_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+CHALLENGER_DIR=$(echo ${SOURCE_DIR%/*/*})
+MONOREPO_DIR=$(echo ${SOURCE_DIR%/*/*/*})
+
+cd $MONOREPO_DIR
+make
+make devnet-clean
+make cannon-prestate
+make devnet-up
+
+DEVNET_SPONSOR="ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+DISPUTE_GAME_PROXY=$(jq -r .DisputeGameFactoryProxy $MONOREPO_DIR/.devnet/addresses.json)
+
+echo "----------------------------------------------------------------"
+echo " Dispute Game Factory at $DISPUTE_GAME_PROXY"
+echo "----------------------------------------------------------------"
+
+L2_OUTPUT_ORACLE_PROXY=$(jq .L2OutputOracleProxy $MONOREPO_DIR/.devnet/addresses.json)
+L2_OUTPUT_ORACLE_PROXY=$(echo $L2_OUTPUT_ORACLE_PROXY | tr -d '"')
+
+echo "----------------------------------------------------------------"
+echo " L2 Output Oracle Proxy at $L2_OUTPUT_ORACLE_PROXY"
+echo "----------------------------------------------------------------"
+
+BLOCK_ORACLE_PROXY=$(jq .BlockOracle $MONOREPO_DIR/.devnet/addresses.json)
+BLOCK_ORACLE_PROXY=$(echo $BLOCK_ORACLE_PROXY | tr -d '"')
+
+echo "----------------------------------------------------------------"
+echo " Block Oracle Proxy at $BLOCK_ORACLE_PROXY"
+echo "----------------------------------------------------------------"
+
+CHARLIE_ADDRESS="0xF45B7537828CB2fffBC69996B054c2Aaf36DC778"
+CHARLIE_KEY="74feb147d72bfae943e6b4e483410933d9e447d5dc47d52432dcc2c1454dabb7"
+
+MALLORY_ADDRESS="0x4641c704a6c743f73ee1f36C7568Fbf4b80681e4"
+MALLORY_KEY="28d7045146193f5f4eeb151c4843544b1b0d30a7ac1680c845a416fac65a7715"
+
+echo "----------------------------------------------------------------"
+echo " - Fetching balance of the sponsor"
+echo " - Balance: $(cast balance 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266)"
```

Comment:
Slight nit: this could get out of sync if the `$DEVNET_SPONSOR` changes, could do `cast wallet address --private-key` altho it doesn't output a nice format that is easy to parse, would need to string parse with `awk` or something else

Reply:
Since this has been used and likely will for a while, let's move this out of this PR since it's quite out of scope and I don't see a good minimal solution.